src/interfaces/AutonomyThread.hpp

```diff
@@ -135,7 +135,7 @@ class AutonomyThread
             m_bStopThreads = false;
 
             // Submit single task to pool queue and store resulting future. Still using pool, as it's scheduling is more efficient.
-            std::future<void> fuMainReturn = m_thMainThread.submit(&AutonomyThread::RunThread, this, std::ref(m_bStopThreads));
+            std::future<void> fuMainReturn = m_thMainThread.submit_task([this]() { RunThread(m_bStopThreads); });
```

Comment:
Change to this->RunThread for consistency.

Reply:
Like this?
![image](https://github.com/MissouriMRDT/RoveComm_CPP/assets/43173458/c44bb76e-d9cb-4d33-86db-d91fb235c4a3)